dcrtweetbot.go

```diff
@@ -22,6 +22,8 @@ var shell *ipfs.Shell
 var client *twitter.Client
 var config *Config
 var dcrtimeHost string
+var timestampedTweets = make([]tweetResult, 100)
+var numberOfTweetsVisible = 100
```

Comment:
@fernandoabolafio should this also be part of config ?

Reply:
I don’t think it is needed. Let’s just make it a constant and it should be enough for now.